docs/components-progress-bar.md

```diff
@@ -0,0 +1,92 @@
+# Components
+
+This theme ships some components (as twig macros) that hide the complexity of rendering the same elements over and over again with the correct HTML.
+
+## ProgressBar (single)
+
+Progress Bar has been implemented to simplify the use of the Tabler Progress Bar component.
+`ProgressBar()` is a Shortcut of `ProgressBars()` with an array of ONE item.
```

Comment:
Can we use `progress_bar` and `progress_bars` please.

We already have `status_dot` and `status_indicator`.

Reply:
Sure 👍